File: Bruteforce/Boj2003/석희.cpp
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class BOJ_2003 {
    static int n, m;
    static int[] arr;

    public static void main(String[] args) throws IOException {

        input();

        StringBuilder sb = new StringBuilder();
        sb.append(solve());
        System.out.println(sb);
    }

    static void input() throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st;

        //region n, m
        st = new StringTokenizer(br.readLine());
        n = Integer.parseInt(st.nextToken());
        m = Integer.parseInt(st.nextToken());
        //endregion

        //region arr
        arr = new int[n];
        st = new StringTokenizer(br.readLine());
        for(int i = 0; i < n; i++) {
            arr[i] = Integer.parseInt(st.nextToken());
        }
        //endregion
    }

    static int solve() {
        int answer = 0;
        int sum = 0;
        for(int i = 0; i < n; i++) {
            sum = 0;
            // 숫자 하나만으로 m이 될 수 있으니까 i부터 확인
            for(int j = i; j < n; j++) {
                sum += arr[j];
                if(sum == m) answer++;
            }
        }
        return answer;
    }
}

File: Bruteforce/Boj1476/석희.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class BOJ_1476 {
    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());

        int E = Integer.parseInt(st.nextToken());
        int S = Integer.parseInt(st.nextToken());
        int M = Integer.parseInt(st.nextToken());

        int year = 1;
        int e = 1, s= 1, m = 1;
        while (!(e == E && s == S && m == M)) {
            e++;
            s++;
            m++;
            year++;

            if(e == 16) e = 1;
            if(s == 29) s = 1;
            if(m == 20) m = 1;
        }

        StringBuilder sb = new StringBuilder();
        sb.append(year);
        System.out.println(year);
    }
}
